src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,268 @@
+---
+title: Filter panel
+description: The filter panel is a component that can be customized to provide categorized filters to help users trim down large amounts of content, data and/or search results.
+---
+
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentFooter from 'components/ComponentFooter';
+
+<ComponentDescription name="Filter panel" type="ui" />
+
+<AnchorLinks>
+
+<AnchorLink>Default</AnchorLink>
+<AnchorLink>Anatomy</AnchorLink>
+<AnchorLink>Selection methods</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
+<AnchorLink>Category guidance</AnchorLink>
+<AnchorLink>Design and functional specifications</AnchorLink>
+<AnchorLink>Development documentation</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+
+## Default
+
+Filter panel includes a heading, configurable filters and an optional children heading area. Filter panel supports multiple categories and selection methods, it is typically placed on the left side of the page.
+
+<Row>
+
+<Column colMd={4} colLg={6}>
+
+![A visual example of the Filter panel component](../../images/component/filter-panel/filter-panel-default.jpg)
+
```

Comment:
A caption (and also for the other images, except for anatomy) would be great. It's a partial sentence here so no need for a period at the end.

> Example of the filter panel component